content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -130,11 +130,29 @@ Finally, run the following commands:
 
 ### APM and DogStatsD
 
-To configure DogstatsD clients and APM tracers automatically, inject the environment variables `DD_AGENT_HOST` and `DD_ENTITY_ID` by using one of the following:
+To configure and inject APM tracers libraries and DogstatsD clients automatically:
```

Comment:
This used to say "do one of the following". Is it still that, or do they have to do both?